file.go

```diff
@@ -590,9 +525,9 @@ func (f *File) Undo(isundo bool) (q0, q1 int, ok bool) {
 			f.mod = u.mod
 			f.treatasclean = false
 			f.b.Insert(u.p0, u.buf)
-			for _, text := range f.text {
-				text.inserted(u.p0, u.buf)
-			}
+			f.oeb.AllObservers(func(i interface{}) {
```

Comment:
Why not just call `oeb.inserted`?

Reply:
Oeb doesn't have an inserted, BufferObserver does, so I'd have to loop throught the map using AllObservers to run inserted on them anyways.

Reply:
Please add one where the implementation of `ObservableEditableBuffer.inserted` will then contain the `for` loop. Create a similar `ObservableEditableBuffe.deleted`. Does this make sense?